portal-parent/ozwillo-portal-front/src/main/resources/application.yml

```diff
@@ -132,3 +132,11 @@ cache:
   hosts:
     - 192.168.56.1
     - 192.168.56.102
+
+# Be sure the MTA have STARTTLS authentification on 587 port if you use it
```

Comment:
Would be even better to use "smtps" instead of STARTTLS.

FWIW, [this code](https://github.com/ozwillo/ozwillo-kernel/blob/ea1924d7ae0bf326c587872ef5e3bb13a3778c9b/oasis-webapp/src/main/java/oasis/mail/MailModule.java#L122-L147)'s uses SMTPS in production (and plain-text SMTP on localhost in dev – through the [default configuration](https://github.com/ozwillo/ozwillo-kernel/blob/ea1924d7ae0bf326c587872ef5e3bb13a3778c9b/oasis-webapp/src/main/resources/reference.conf#L32-L35))